### DISCUSSION TITLE
Open Mindmap Dialog box - ability to search a map by typing in the dialog box

### DISCUSSION THREAD

**Discussion by manasmbellani** [freeplane/freeplane#751] [Help]
Hi, Is it possible to find a mindmap in the Freeplane Open Dialog box by typing for the keywords in Freeplane in Macbook ? I am using a Macbook Air (Mac OS Monterey 12.6) and when I select the File > Open "Mind Map" option, I am not able to type the name of the map to find it like I can in a Finder view. 

I can only scroll down a list of the mind maps which can be inefficient if I have large number of maps in a folder.

Was hoping there was an option to be able to achieve this in Freeplane. As a workaround, I have been using the "Open current map directory option" which allows me to type in mind map name in the Finder window and highlights it for me in the folder. But I was hoping there would be a way to do this inside Freeplane as sometimes the same map can open twice via Finder.

**Comment by euu2021** [comment]
>[...] sometimes the same map can open twice via Finder

Do you mean, it opens on a second instance of FreePlane? You can disable it at `Preferences…->Environment->Single program instance->Open files in a running instance`
